spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/registry/DataFlowAppRegistryPopulator.java

```diff
@@ -27,34 +30,39 @@
  *
  * @author Mark Fisher
  */
-public class DataFlowUriRegistryPopulator implements InitializingBean {
+public class DataFlowAppRegistryPopulator implements InitializingBean, ResourceLoaderAware {
```

Comment:
As discussed offline, no you're not. This class is only used in tests and has always been